templates/company/pscs/list.html.tx

```diff
@@ -119,7 +119,7 @@
                         </span>
                         % if $psc.ceased_on || $psc.ceased {
                             <span id="psc-status-tag-<% $~psc.count %>" class="status-tag ceased-tag font-xsmall">Ceased</span>
-                        % } else if $company.company_status != 'dissolved' && $company.company_status != 'converted-closed'{
+                        % } else if $company.company_status != 'dissolved' && $company.company_status != 'converted-closed' && $company.company_status != 'removed'{
```

Comment:
I see you are just stopping it displaying `Active`, not getting it to display `Ceased`. I was going to get it to display `Ceased` in my case, but that was just a feeling from looking at the code. Do you know what it should do?